packages/database/src/modelClasses/MapOverlay.js

```diff
@@ -12,7 +12,20 @@ class MapOverlayType extends DatabaseType {
 }
 
 export class MapOverlayModel extends DatabaseModel {
+  notifiers = [onChangeDeleteRelation];
+
   get DatabaseTypeClass() {
     return MapOverlayType;
   }
 }
+
+const onChangeDeleteRelation = async ({ type: changeType, record }, models) => {
```

Comment:
This notifier is for removing any orphan relations in case devs forget when they remove mapOverlays/mapOverlayGroups (context: https://github.com/beyondessential/tupaia/pull/1074#discussion_r466104407).